cs/di-configuration.texy

```diff
@@ -308,6 +308,24 @@ $database = $container->getByType(PDO::class);
 
 
 
+Řetězová syntaxe
+----------------
+
+Volání method lze pohodlně zřetězit za sebe pomocí zápisu s dvěma dvojtečkami (`::`). Zápis lze aplikovat na `class`, `factory` i `setup`.
+
+/--neon
+services:
+	- Factory::create( @service::abc() )::another( Class() )::last()
+
+	foo:
+		factory: Factory
+		setup:
+			- setCache
+			- set( @service::abc() )::last()
+\--
+
+
+
```

Comment:
Co to nazvat "Pokročilá syntaxe" a spojit to s druhou polovinou toho, co je v `Propojení služeb`?

Reply:
Zkusil jsem, pripravil jsem CZ i EN verzi.